build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleReleaseNotesPlugin.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 the original author or authors.
+ * Copyright 2025 the original author or authors.
```

Comment:
❔  Is it necessary to change the copyright year?
I thought, we agreed that we add it on file creation and don't change it later

Reply:
That is done automatically with IntelliJ since I messed with the settings. I removed it.